etna/insights/models.py

```diff
@@ -46,12 +70,30 @@ class InsightsPage(HeroImageMixin, TeaserImageMixin, BasePage):
 
     sub_heading = models.CharField(max_length=200, blank=False)
     body = StreamField(InsightsPageStreamBlock, blank=True, null=True)
-
+    topic = models.ForeignKey(
+        "collections.TopicExplorerPage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    time_period = models.ForeignKey(
+        "collections.TimePeriodExplorerPage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    tags = ClusterTaggableManager(through=TaggedInsights, blank=True)
     content_panels = (
         BasePage.content_panels
         + HeroImageMixin.content_panels
         + [
             FieldPanel("sub_heading"),
+            PageChooserPanel("featured_insight"),
```

Comment:
I think this needs to be added to ``InsightsIndexPage.content_panels``, rather than ``InsightsPage.content_panels`` ?

Reply:
Yes , the relationship is in the InsightsIndexPage via featured_insight foreign field, so the pagechooserpanel must be visible on the InsightsIndexPage admin page.